src/Moq/Protected/IProtectedMock.cs

```diff
@@ -140,6 +140,20 @@ public interface IProtectedMock<TMock> : IFluentInterface
 		/// remember to use <see cref="ItExpr"/> rather than <see cref="It"/>.</param>
 		void Verify(string methodName, Times times, params object[] args);
 
+		/// <summary>
+		/// Specifies a verify for a void method with the given <paramref name="methodName"/>,
+		/// optionally specifying arguments for the method call. Use in conjunction with the default
+		/// <see cref="MockBehavior.Loose"/>.
+		/// </summary>
+		/// <exception cref="MockException">The invocation was not call the times specified by
+		/// <paramref name="times"/>.</exception>
+		/// <param name="methodName">The name of the void method to be verified.</param>
+		/// <param name="times">The number of times a method is allowed to be called.</param>
+		/// <param name="exactParameterMatch">Should the parameter types match exactly types that were provided</param>
+		/// <param name="args">The optional arguments for the invocation. If argument matchers are used, 
+		/// remember to use <see cref="ItExpr"/> rather than <see cref="It"/>.</param>
+		void Verify(string methodName, Times times, bool exactParameterMatch, params object[] args);
```

Comment:
Some thinking aloud below; no need for you to change anything.

If we placed `exactParameterMatch` before `times`, there would be less chance of a breaking change (in case where someone currently passes a `bool` as the first argument; see also the description of #751). On the other hand, it makes more sense to place all argument-related parameters next to each other instead of having them artificially separated by `times`.

The way you currently seems logical to me. And since Moq's `Protected`-APIs probably aren't the busiest ones we can probably risk introducing a breaking change here.

Reply:
I initially put it before `times` but then I didn't like it and moved it to before `args`, I'd be more than happy to change it if you change your mind later on.

Reply:
I think i need to resolve this conversation for the change request to be complete on GitHub, I maybe wrong, please unresolve it if you change your mind.